docs/options api.md

```diff
@@ -269,7 +269,8 @@ placed on them.
 
 ### PriorityLocations
 Marks locations given here as `LocationProgressType.Priority` forcing progression items on them if any are available in
-the pool.
+the pool. Progression items without a deprioritized flag will be used first when filling priority_locations.  Progression items with
```

Comment:
Nitpick af: There's a double space between "priority_location." and "Progression items"